api/v1/kusk.yaml

```diff
@@ -0,0 +1,1147 @@
+              description: |
+                script content type can be:  - direct content - created from file,  - git repo directory checkout in case when test is some kind of project or have more than one file,
+              enum:
+              - content
+              - git
+              type: string
+            name:
+              description: script name - Custom Resource name - must be unique, use only
+                lowercase numbers and dashes (-)
+              example: kubeshop-homepage-test
+              type: string
+            namespace:
+              description: kubernetes namespace (defaults to 'testkube')
+              example: testkube
+              type: string
+            repository:
+              $ref: '#/components/schemas/Repository'
+            type:
+              description: script type - what executor type should be used during test
+                execution
+              example: postman/collection
+              type: string
+          type: object
+        ServerInfo:
+          description: Server information with build version, build commit etc.
+          properties:
+            commit:
+              description: build commit
+              type: string
+            version:
+              description: build version
+              type: string
+          required:
+          - version
+          type: object
+    externalDocs:
+      description: Find out more about testkube
+      url: http://testkube.io
+    info:
+      contact:
+        email: testkube@kubeshop.io
+      description: TestKube provides a Kubernetes-native framework for test definition,
+        execution and results
+      license:
+        name: MIT
+        url: https://opensource.org/licenses/MIT
+      title: TestKube API
+      version: 1.0.0
+    openapi: 3.0.1
+    paths:
+      /executions:
+        get:
+          description: Returns array of scripts executions
+          operationId: listExecutions
+          parameters:
+          - $ref: '#/components/parameters/ExecutionsPageSize'
+          - $ref: '#/components/parameters/ExecutionsPageIndex'
+          - $ref: '#/components/parameters/ExecutionsStatusFilter'
+          - $ref: '#/components/parameters/ExecutionsStartDateFilter'
+          - $ref: '#/components/parameters/ExecutionsEndDateFilter'
+          responses:
+            "200":
+              content:
+                application/json:
+                  schema:
+                    $ref: '#/components/schemas/ExecutionsResult'
+              description: successful operation
+            "500":
+              content:
+                application/problem+json:
+                  schema:
+                    items:
+                      $ref: '#/components/schemas/Problem'
+                    type: array
+              description: problem with getting script executions from storage
+          summary: Get all script executions
+          tags:
+          - executions
+          - api
```

Comment:
Another entry for dashboard to work (it prepends / to the api path), as we need to add /executions/ route:
```
/executions/:
        get:
          description: Returns array of scripts executions
          operationId: listExecutionsA
          parameters:
          - $ref: '#/components/parameters/ExecutionsPageSize'
          - $ref: '#/components/parameters/ExecutionsPageIndex'
          - $ref: '#/components/parameters/ExecutionsStatusFilter'
          - $ref: '#/components/parameters/ExecutionsStartDateFilter'
          - $ref: '#/components/parameters/ExecutionsEndDateFilter'
          responses:
            "200":
              content:
                application/json:
                  schema:
                    $ref: '#/components/schemas/ExecutionsResult'
              description: successful operation
            "500":
              content:
                application/problem+json:
                  schema:
                    items:
                      $ref: '#/components/schemas/Problem'
                    type: array
              description: problem with getting script executions from storage
          summary: Get all script executions
          tags:
          - executions
          - api
```

Reply:
@exu do you have the file you used to do this? if so can you add the change so i don't break this

Reply:
@Tarick I changed it can you please take another look